lib/resolver.js

```diff
@@ -351,9 +351,7 @@ class SchemaResolver {
     
         if (Util.isUndefined(schema.minLength)) {
             schema.minLength = 0;
-        }
-    
-        if (schema.minLength === 0) {
+        } else if (schema.minLength === 0) {
```

Comment:
New line here please to keep with rest of style.

Reply:
Fixed that.
And also linted the file, which removed few spaces on empty lines. I hope that's ok.